src/assets/scripts/client/InputController.js

```diff
@@ -451,7 +451,8 @@ export default class InputController {
                 this.closeAllDialogs();
 
                 if (!_includes(currentCommandInputValue, this.input.callsign) ||
-                    currentCommandInputValue.trim() === this.input.callsign
+                    currentCommandInputValue.trim() === this.input.callsign ||
+                    this.input.callsign === ''
```

Comment:
For readability, could we break these three conditions out into clearly named consts, and then do the if on the consts, instead of inline?

Reply:
Done in 74d768a